jekyll/_cci2_ja/caching-strategy.md

```diff
@@ -16,6 +16,15 @@ version:
 * TOC
 {:toc}
 
+## キャッシュストレージのカスタマイズ
+{: #caching-and-self-hosted-runner }
+
+セルフホストランナーを使用する場合、プランに含まれるネットワークとストレージ使用量には制限があります。 キャッシュに関連するアクションには、ネットワークとストレージの使用が発生するものがあります。 お客様の使用量が制限を超えた場合、料金が発生します。
+
+キャッシュを長期間保存すると、ストレージコストに影響が及ぶため、キャッシュを保存する理由やユースケースに応じた必要なキャッシュの保存期間を明確にすることをお勧めします。 ニーズに合う場合は、キャッシュのストレージ保存期間を短く設定し、コストを削減しましょう。
+
+[CircleCI Web アプリ](https://app.circleci.com/)で **Plan > Usage Controls** に移動し、キャッシュのストレージ使用量や保存期間をカスタマイズすることができます。 ネットワークとストレージ使用量の管理の詳細については、[データの永続化]({{site.baseurl}}/2.0/persist-data/)のページを参照してください。
```

Comment:
`{{site.baseurl}}/ja/2.0/persist-data/` to add `ja` before `2.0` for better experience.